CHANGELOG.develop

```diff
@@ -1599,6 +1640,7 @@ Benner and Paweł Siudak):
 - Key bindings:
   - ~SPC m E e~ is for =tide-fix= which was originally added as ~SPC m r f~
     (thanks to Simon Altschuler)
+  - ~SPC m E d~ to add =tslitn-disable-next-line= at point (thanks to Daniel Perez Alvarez)
```

Comment:
Typo: "tslitn" should be "tslint". ...Although I see that the documentation has the same typo.